google/cloud/storage/tests/curl_resumable_upload_session_integration_test.cc

```diff
@@ -82,7 +82,7 @@ TEST_F(CurlResumableUploadIntegrationTest, WithReset) {
   std::string const contents(UploadChunkRequest::kChunkSizeQuantum, '0');
   StatusOr<ResumableUploadResponse> response =
       (*session)->UploadChunk(contents, 2 * contents.size());
-  ASSERT_TRUE(response.status().status_code() == 200 or response.status().status_code() == 308)
+  ASSERT_TRUE(response.status().ok())
```

Comment:
Join lines?

Reply:
Why, yes.  I applied clang-format manually, I do not understand why it passed the clang-format checks.